app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -342,6 +341,19 @@ class PipelineViz extends React.Component {
       );
     });
 
+    stageContainers = stageContainers.reduce((containers, stage, i) => {
```

Comment:
nit: you can just call reduce on the output of the map and make `stageContainers` const again.